clients/src/main/java/org/apache/kafka/clients/consumer/internals/ConsumerCoordinator.java

```diff
@@ -1352,8 +1370,22 @@ public void handle(OffsetCommitResponse commitResponse, RequestFuture<Void> futu
             Set<String> unauthorizedTopics = new HashSet<>();
 
             for (OffsetCommitResponseData.OffsetCommitResponseTopic topic : commitResponse.data().topics()) {
+                String topicName = topic.name();
+
+                if (commitResponse.version() >= 9) {
```

Comment:
@Hangleton I had a deeper look into this and it seems that we could get the version with `this.response.requestHeader().apiVersion()`. Could you check if this would work?

Reply:
Yes, this works. Thanks for the call-out.